liquibase-core/src/main/java/liquibase/integration/commandline/Main.java

```diff
@@ -886,20 +889,27 @@ private void addWarFileClasspathEntries(File classPathFile, List<URL> urls) thro
         URL url = new URL("jar:" + classPathFile.toURI().toURL() + "!/WEB-INF/classes/");
         logger.info("adding '" + url + "' to classpath");
         urls.add(url);
-        JarFile warZip = new JarFile(classPathFile);
-        Enumeration<? extends JarEntry> entries = warZip.entries();
-        while (entries.hasMoreElements()) {
-            JarEntry entry = entries.nextElement();
-            if (entry.getName().startsWith("WEB-INF/lib")
-                    && entry.getName().toLowerCase().endsWith(".jar")) {
-                File jar = extract(warZip, entry);
-                URL newUrl = new URL("jar:" + jar.toURI().toURL() + "!/");
-                logger.info("adding '" + newUrl + "' to classpath");
-                urls.add(newUrl);
-                jar.deleteOnExit();
+        JarFile warZip = null;
+        try {
+            warZip = new JarFile(classPathFile);
+            Enumeration<? extends JarEntry> entries = warZip.entries();
+            while (entries.hasMoreElements()) {
+                JarEntry entry = entries.nextElement();
+                if (entry.getName().startsWith("WEB-INF/lib")
+                        && entry.getName().toLowerCase().endsWith(".jar")) {
+                    File jar = extract(warZip, entry);
+                    URL newUrl = new URL("jar:" + jar.toURI().toURL() + "!/");
+                    logger.info("adding '" + newUrl + "' to classpath");
+                    urls.add(newUrl);
+                    jar.deleteOnExit();
+                }
+            }
+        }finally {
+            if(warZip != null) {
```

Comment:
don't need the close

Reply:
`warZip` creation should be wrapped `with try with resources`